release_notes/ocp-4-1-release-notes.adoc

```diff
@@ -0,0 +1,863 @@
+cluster` commands, Minishift, and CDK. {product-title} 4.1 focuses on ease of
+access and native experience, with a native installer on macOS and Microsoft
+Windows, native hypervisor support, and tray icon integration.
+
+[[ocp-4-1-registry]]
+=== Registry
+
+[[ocp-4-1-registry-managed-by-operator]]
+==== The registry is now managed by an Operator
+
+The registry is now managed by an Operator instead of `oc adm registry`.
+
+[[ocp-4-1-networking]]
+=== Networking
+
+[[ocp-4-1-cno]]
+==== Cluster Network Operator (CNO)
+
+The cluster network is now configured and managed by an Operator. The Operator
+upgrades and monitors the cluster network.
+
+[[ocp-4-1-openshift-sdn]]
+==== OpenShift SDN
+
+The default mode is now `NetworkPolicy`.
+
+[[ocp-4-1-multus]]
+==== Multus
+
+Multus is a meta plug-in for Kubernetes Container Network Interface (CNI), which
+enables a user to create multiple network interfaces per pod.
+
+[[ocp-4-1-web-console]]
+=== Web console
+
+[[ocp-4-1-developer-catalog]]
+==== Developer Catalog
+
+{product-title} 4.1 features a redesigned Developer Catalog that brings all of
+the new Operators and existing broker services together, with new ways to
+discover, sort ,and understand how to best use each type of offering. The
+Developer Catalog is the entry point for a developer to access all services
+available to them. It merges all capabilities from Operators, the Service
+Catalog, brokers, and Source-to-Image (S2I).
+
+[[ocp-4-1-operator-powered-UI-extensions]]
+==== Operator-powered user interface extensions
+
+Now with Operators, you can pick and choose what functionality you want to
+enable, allowing the user to customize the interface.
+
+[[ocp-4-1-new-management-screens]]
+==== New management screens
+
+New management screens in {product-title} 4.1  support automated operations.
+Examples include the management of taints, tolerations, and cluster settings.
+
+[[ocp-4-1-security]]
+=== Security
+
+In {product-title} 4.1, an Operator is utilized to install, configure, and manage
+the certificate signing server. Certificates are managed as secrets, and are
+stored and encrypted in etcd.
+
+[[ocp-4-1-notable-technical-changes]]
+== Notable technical changes
+
+{product-title} 4.1 introduces the following notable technical changes.
+
+[discrete]
+[[ocp-4-1-buildah]]
+==== All image builds are performed by buildah
+
+All image builds are now performed by buildah instead of the docker daemon.
+
+[discrete]
+[[ocp-4-1-SecurityContextConstraints]]
+==== SecurityContextConstraints
+
+`SecurityContextConstraints` now only exist in the `security.openshift.io` group.
+
+[discrete]
+[[ocp-4-1-Service-CA-bundle]]
+==== Service CA bundle changes
+
+Pods can trust cluster-created certificates, which are only signed for internal
+DNS names, by using a CA bundle that is automatically injected into any
+configMap annotated with `service.alpha.openshift.io/inject-cabundle=true`. The
```

Comment:
@enj this is beta not alpha right?

also should probably note that this annotation results in wiping out existing content in the configmap.